packages/slate/test/interfaces/Editor/unhangRange/void-hanging.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx jsx */
+import { Editor } from 'slate'
+import { jsx } from '../../..'
+
+export const input = (
+  <editor>
+    <block>
+      <anchor />
+      This is a first paragraph
+    </block>
+    <block>This is the second paragraph</block>
+    <block void>
+      <focus />
+    </block>
+  </editor>
+)
+
+export const test = editor => {
+  return Editor.unhangRange(editor, editor.selection)
+}
+
+export const output = {
+  anchor: { path: [0, 0], offset: 0 },
+  focus: { path: [0, 0], offset: 25 },
+}
```

Comment:
This doesn't look right: it completely skipped the second paragraph. I am going to fix this with a new PR.